build.ts

```diff
@@ -67,8 +68,27 @@ try {
 
 	const isDebugBuild = argv.includes('--debug');
 	const buildType = isDebugBuild ? 'development' : 'production';
+	let platform: string = 'null' ;
+	let extension: string = 'null' ;
+	switch(process.platform) {
```

Comment:
I propose to tag this PR as draft, until bun support it https://twitter.com/jarredsumner/status/1661685547004477440?s=20

Reply:
No, that could be months away and this branch is something we want to ship immediately. The changes I mentioned above are perfectly sufficient, control the build via an argument and we'll continue to build it on Linux. We would do this on our CI anyway since Linux runners are cheaper and cross-compilation was always the goal.

Reply:
I revert to build only for Windows